tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -114,8 +117,13 @@ def type_comment(self, value):
         element.send_keys(value)
 
     def tap_comment(self):
-        self.marionette.find_element(*self._add_new_note_locator).tap()
+        element = self.marionette.find_element(*self._add_new_note_locator)
+        element.tap()
+        self.marionette.execute_script(
+            'arguments[0].scrollIntoView(true);', [element])
```

Comment:
This looks good, but I thought this method is already there somewhere in gaiatest already, because I remember seeing something like this.  If not, we should definitely put it into the base class definition.